README.md

```diff
@@ -1,13 +1,47 @@
-# qiskit-dynamics
+# Qiskit Dynamics
 
 [![License](https://img.shields.io/github/license/Qiskit/qiskit-experiments.svg?style=popout-square)](https://opensource.org/licenses/Apache-2.0)
 
-**This repo is still under active development, there will be breaking API changes**
+**This repo is still in the early stages of development, there will be breaking API changes**
 
-**Qiskit** is an open-source framework for working with noisy quantum computers
-at the level of pulses, circuits, and algorithms.
+Qiskit Dynamics is an open-source project for building, transforming, and solving
+models of quantum systems in Qiskit.
+
+The goal of Qiskit Dynamics is to provide access to different numerical
+methods, and to automate common processes typically performed by hand,
+e.g. entering rotating frames, or doing the rotating wave approximation.
+
+Qiskit Dynamics can be configured to use either
+[NumPy](https://github.com/numpy/numpy) or [JAX](https://github.com/google/jax)
+as the backend for array operations. [NumPy](https://github.com/numpy/numpy) is the default,  
+and [JAX](https://github.com/google/jax) is an optional dependency.
+[JAX](https://github.com/google/jax) provides just-in-time compilation, automatic differentiation,
+and GPU execution, and therefore is well-suited to tasks involving repeated
+evaluation of functions with different parameters; E.g. simulating a model of a quantum system
+over a range of parameter values, or optimizing the parameters of control sequence.
+
+## Installation
+
+Qiskit Dynamics may be installed using pip via:
+
+```
+pip install qiskit-dynamics
+```
+
+Additionally, Qiskit Dynamics may be installed simultaneously with the CPU version of
+JAX via:
+
+```
+pip install "qiskit-dynamics[jax]"
+```
+
+which is equivalent to
```

Comment:
Lines 38-43 aren't really necessary and could be removed. Instead could add a sentence like 
> Installing JAX with GPU support must be done manually, for instructions refer to the jax install documentation (link).

Reply:
Added (with link).